frontend/app/src/store/settings/frontend.ts

```diff
@@ -138,7 +140,7 @@ export const useFrontendSettingsStore = defineStore('settings/frontend', () => {
     refreshPeriod,
     renderAllNftImages,
     savedFilters,
-    selectedTheme,
+    subscriptDecimals,
```

Comment:
you also need to run `pnpm run lint:fix` after the rebase here the only changes should be this line and the one adding the const